lib/static_sync/processor.rb

```diff
@@ -46,6 +46,8 @@ def handle_conflict(file)
         if file.cached?
           raise ConflictError, "modifications to existing cached files are not allowed."
```

Comment:
@fraserxu @nemski Maybe we should change this config: https://github.com/envato/marketplace/blob/master/lib/tasks/assets.rake#L102